_data/committers.yml

```diff
@@ -256,6 +256,10 @@
   role: Committer
   alias: sunchao
   affiliation: Uber
+- name: Curt Hagenlocher
+  role: Committer
+  alias: curth
```

Comment:
This is the apache alias (not the github alias)

Reply:
Looks good